.rubocop.yml

```diff
@@ -28,6 +28,7 @@ Metrics/LineLength:
     - 'app/models/concerns/file_set/image_file_behavior.rb'
     - 'app/models/concerns/file_set/raster_file_behavior.rb'
     - 'app/models/concerns/file_set/vector_file_behavior.rb'
+    - 'app/models/concerns/curation_concerns/file_set/derivatives.rb'
```

Comment:
I'm sympathetic to most of these rubocop exclusions, but can you reformat `derivatives.rb` to have shorter lines?  The long lines all look like hashes that could be easily split, and it would make the code much more readable.